lib/helpers.js

```diff
@@ -132,9 +132,29 @@ function renameCache(source, destination, options) {
   });
 }
 
+
+function renameCacheAndCleanUp(source, destination, options) {
+  debug('Renaming cache: [' + source + '] to [' + destination + ']', options);
+
+  return caches.keys().then(function(cacheNames) {
+    Promise.all(
+
+      cacheNames.map(function(cacheName) {
+        if (destination !== cacheName && source !== cacheName) {
+          return caches.delete(cacheName);
```

Comment:
This seems dangerous!

If a user is either managing a cache outside of `sw-toolbox` (e.g. via `sw-precache` or just vanilla Cache Storage API calls) or using `sw-toolbox` but with explicitly named caches (e.g. in [this example](https://github.com/GoogleChrome/sw-toolbox/blob/master/recipes/cache-expiration-options/service-worker.js#L18)), then those caches would end up being deleted by this code.